shell/ci/release/docker.sh

```diff
@@ -173,7 +173,7 @@ build_and_push_image() {
   fi
   (
     set -x
-    docker buildx build "${args[@]}"
+    docker buildx --builder devbase build "${args[@]}"
```

Comment:
I just realized L34 of `buildx.sh` scripts the builder creation on macOS (because it isn't required). we'd probably have to encapsulate that logic here as well (omit the flag on macOS).

Reply:
I would just conditionally append to the `args` associative array.

Reply:
(or, if positional, make a new array to pass to it, to keep `set -x` from being noisy)